source/main/ipc-api/kill-process.js

```diff
@@ -1,10 +1,10 @@
 // @flow
-import { ipcMain } from 'electron';
+import { ipcMain, app } from 'electron';
 
 export default () => {
   ipcMain.on('kill-process', () => {
+    app.quit();
     // if (event.sender !== mainWindow.webContents) return;
-    // TODO: fix this
-    process.exit(20);
```

Comment:
@MarcusHurney please be careful with this change - exit code 20 might be necessary for cardano-launcher … @cleverca22 can you take a look at this?

Reply:
@DominikGuzei 
Changing this line to **app.quit** ended up being unnecessary so I've changed it back. All tests still pass.